src/app/src/test/kotlin/org/imperial/mrc/hint/integration/ErrorReportTests.kt

```diff
@@ -0,0 +1,52 @@
+package org.imperial.mrc.hint.integration
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import org.assertj.core.api.Assertions
+import org.imperial.mrc.hint.ConfiguredAppProperties
+import org.imperial.mrc.hint.clients.FuelFlowClient
+import org.imperial.mrc.hint.controllers.ErrorReportController
+import org.imperial.mrc.hint.models.ErrorReport
+import org.imperial.mrc.hint.models.Errors
+import org.junit.jupiter.api.Test
+import java.time.Instant
+
+class ErrorReportTests: SecureIntegrationTests()
+{
+    @Test
+    fun `can post error report`()
```

Comment:
This is testing the the FuelFlowClient and controller classes together, but I was suggesting that you add a test of the new endpoint so it would test both these things and also the ErrorReport deserialisation. So I'd expect it to look more like the other integration tests at this level, using `testRestTemplate.postForEntity` e.g. [ModelRunTests](https://github.com/mrc-ide/hint/blob/92af71ff8273ed38215efaf15601ddf9727e904c/src/app/src/test/kotlin/org/imperial/mrc/hint/integration/ModelRunTests.kt#L55)